
Solaris has not "fmemopen" so we just disable this for now


--- samba-4.10.4_manuell/libcli/dns/wscript_build	Sa. Jun 15 17:18:45 2019
+++ samba-4.10.4/libcli/dns/wscript_build	So. Jun 16 00:10:37 2019
@@ -5,10 +5,6 @@
         public_deps='LIBTSOCKET tevent-util ndr-standard',
         private_library=True)
 
-bld.SAMBA_BINARY('resolvconftest',
-                  source='resolvconftest.c',
-                  deps='clidns',
-                  install=False)
 
 bld.SAMBA_SUBSYSTEM('dns_lookup',
                     source='dns_lookup.c',
